mobile/lib/components/newest_recording/newest_recording_widget.dart

```diff
@@ -18,27 +18,30 @@ class NewestRecordingWidget extends StatelessWidget {
     final defaultShadowColor = context.colors.highlightGreen;
 
     return SwipeableCard(
-      // TODO: Change this for array of real SwipeableCardItems
```

Comment:
Don't remove that TODO, as the issue is not resolved - the `SwipeableCardItems` are still not "real"